rdrf/rdrf/views/xnat_view.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.core.exceptions import PermissionDenied
+from django.http import JsonResponse
+from django.views import View
+
+from rdrf.integration.xnat_service import xnat_experiments_scans
+
+logger = logging.getLogger(__name__)
+
+
+class XnatScansLookup(View):
+
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.is_staff:
```

Comment:
Might be able to use `StaffMemberRequiredMixin` instead?

Reply:
Yes 👍

Reply:
done.